js/samples/advanced-csharp/Pages/Document.cshtml

```diff
@@ -75,7 +75,22 @@
             const token = await getImmersiveReaderTokenAsync(Canary);
             const subdomain = await getImmersiveReaderSubdomainAsync();
 
-            ImmersiveReader.launchAsync(token, subdomain, data);
+            // Learn more about options https://docs.microsoft.com/azure/cognitive-services/immersive-reader/reference#options
+            const options = {
+                cognitiveAppId: 'AdvancedSampleWebApp',
```

Comment:
cognitiveAppId is not an option we expose in the Public SDK, should remove this

Reply:
removed for all html pages